Scripts/check-examples

```diff
@@ -0,0 +1,367 @@
+#! /usr/bin/env bash
```

Comment:
Glad this exists but would prefer not to check it in as dead code without actually hooking it up. Better to cut an issue to follow up on instead.

Reply:
As it is merged, I'll make sure it does not stay dead very long.